### PR TITLE
Added an assembly and modules for expanding the cluster.

### PR DESCRIPTION
**Pull request by johnwilkins**
Fixes: https://issues.redhat.com/browse/OSDOCS-1308
@ahardin-rh 
@rlopez133 
@iranzo 
Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@johnwilkins is this for 4.7?

**Comment by iranzo** [comment]
@johnwilkins will it cloned US once merged or should I?

**Comment by johnwilkins** [comment]
@iranzo I've already got it in the upstream repo and I have an upstream version in section 13.3 of the upstream docs.

**Comment by ahardin-rh** [comment]
Confirmed with @johnwilkins that this is for 4.6

File: modules/ipi-install-preparing-the-bare-metal-node.adoc [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+= Preparing the bare metal node
+
+Preparing the bare metal node requires executing the following procedure from the provisioner node or from a control plane node.
+
+.Procedure
+
+. Get the `oc` binary, if needed. It should already exist on the provisioner node.
++
+----
```
Comment (ahardin-rh) [inline modules/ipi-install-preparing-the-bare-metal-node.adoc:15]: [source,bash]

File: modules/ipi-install-preparing-the-bare-metal-node.adoc [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+[kni@openshift-master-0 ~]$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/openshift-client-linux.tar.gz | tar zxvf - oc
+[kni@openshift-master-0 ~]$ sudo cp oc /usr/local/bin
```
Comment (ahardin-rh) [inline modules/ipi-install-preparing-the-bare-metal-node.adoc:17]: Separate multiple commands out into their own command blocks per our updated style guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+----
+
+. Get the machine set.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc get machinesets -n openshift-machine-api
+----
+
+. Increase the number of worker nodes by 1.
```
Comment (ahardin-rh) [inline modules/ipi-install-provisioning-the-bare-metal-node.adoc:41]: s/1/one  per IBM Style

**Comment by ahardin-rh** [comment]
@johnwilkins Thanks for this! Just some minor style comments from me.

**Pull request merged**